File: CPP/V.2/Food.cpp
#include "Food.h"
#include "Entity.h"
#include <iostream>

Food::Food(const Location& loc, const Vector& vec, const char& model) 
	: Entity(loc, vec, model) {}

void Food::tick()
{
	Entity::tick();
}

void Food::render(int x, int y)
{
	if (Location(x, y) == this->loc)
	{
		std::cout << this->model;
	}
}

Food::~Food()
{

}

File: SnakeCS/Food.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SnakeCS
{
    class Food : Entity
    {
        public Food(Location location, Vector vector, char model) : base(location, vector, model) {}
    }
}
